DataConnectors/CyberArk/CyberArk README.md

```diff
@@ -0,0 +1,46 @@
+# Connect CyberArk to Azure Sentinel 
```

Comment:
Please email us the md file and remove this file from this PR - the documentation for data connector goes to a different repository and should not be in this repo. Thanks.